module/VuFind/src/VuFind/Db/Entity/Comments.php

```diff
@@ -104,4 +105,96 @@ class Comments implements EntityInterface
      * })
      */
     protected $resource;
+
+    /**
+     * Id getter
+     *
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Comment setter
+     *
+     * @param string $comment Comment
+     *
+     * @return Comments
+     */
+    public function setComment(string $comment): Comments
+    {
+        $this->comment = $comment;
+        return $this;
+    }
+
+    /**
+     * Comment getter
+     *
+     * @return string
+     */
+    public function getComment(): string
+    {
+        return $this->comment;
+    }
+
+    /**
+     * Created setter.
+     *
+     * @param Datetime $dateTime Created date
+     *
+     * @return Comments
+     */
+    public function setCreated(DateTime $dateTime): Comments
+    {
+        $this->created = $dateTime;
+        return $this;
+    }
+
+    /**
+     * Created getter
+     *
+     * @return Datetime
+     */
+    public function getCreated(): Datetime
+    {
+        return $this->created;
+    }
+
+    /**
+     * User setter.
+     *
+     * @param null|User $user User that created comment
```

Comment:
You could use `?User` here as more concise syntax (and to match the actual function definition).